libvast/src/system/pipeline_executor.cpp

```diff
@@ -220,15 +232,19 @@ auto pipeline_executor(
         VAST_DEBUG("not all execution nodes are done, waiting");
       }
     } else {
-      VAST_DEBUG("promise ist not pending, discarding down message");
+      VAST_DEBUG("promise is not pending, discarding down message");
     }
   });
   self->state.pipe = std::move(p);
-  return {
-    [self](atom::run) -> caf::result<void> {
-      return self->state.run();
-    },
-  };
+  self->state.allow_unsafe_pipelines
+    = caf::get_or(self->system().config(), "vast.allow-unsafe-pipelines",
+                  self->state.allow_unsafe_pipelines);
+  return {[self](atom::run) -> caf::result<void> {
+            return self->state.run();
+          },
+          [self](atom::run, system::node_actor node) -> caf::result<void> {
+            return self->state.run(std::move(node));
+          }};
```

Comment:
```suggestion
          },};
```

Plus reformatting for readability. This belps a lot with actor behaviors.